notebooks/scarlet-measure.ipynb

```diff
@@ -0,0 +1,2888 @@
+{
```

Comment:
<p>Maybe we should filter out this error since it can get really annoying for users doing a large sequence of measurements? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSSTDESC/BlendingToolKit/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I think this is the error from the ellipticity measurement in the metrics.</p>

Reply:
OK let's ignore for now and revisit later then, at least we have this notebook already which is great